.github/workflows/validator_test.yml

```diff
@@ -0,0 +1,26 @@
+name: Validator tests
+
+on: # yamllint disable-line rule:truthy
+  workflow_call:
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      # checkout source code
+      - name: checkout source
+        uses: actions/checkout@v4.1.1
+      # setup Python environment
+      - name: Set up Python
+        uses: actions/setup-python@v5.1.0
+        with:
+          python-version: "3.12"
```

Comment:
```suggestion
          python-version: "3.11"
```

Reply:
Do we have any hard requirements on `3.11` ?

Reply:
Indeed we do :( .. rhel 9

Reply:
I have pinned the python version in #1573